sycl/doc/GetStartedGuide.md

```diff
@@ -416,10 +416,10 @@ command:
 
     ```bash
     # Install OpenCL FPGA emulation RT
-    # Answer N to clean previous OCL_ICD_FILENAMES configuration
```

Comment:
`install.bat` now requires root privileges, right? Don't we want to support setup thru env var for users who have no such privilege? At least we should mention this requirement.

Reply:
Ability to use `OCL_ICD_FILENAMES` is also important in Python packaging, where DPC++ runtimes are installed in user's Python environment, and `OCL_ICD_FILENAMES` is used to help DPC++ runtime find the sycl CPU device.

Reply:
> `install.bat` now requires root privileges, right? Don't we want to support setup thru env var for users who have no such privilege? At least we should mention this requirement.

The script was always run in the administrative console: https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md?plain=1#L408

Reply:
> Ability to use `OCL_ICD_FILENAMES` is also important in Python packaging, where DPC++ runtimes are installed in user's Python environment, and `OCL_ICD_FILENAMES` is used to help DPC++ runtime find the sycl CPU device.

It is not what was introduced by the change. It is reaction to the change in OpenCL RT which ignores OCL_ICD_FILENAMES configuration in the administrative console. For regular usage it is more robust to use registry keys. If you need environment configuration you can still use OCL_ICD_FILENAME in non-administrative console.

Reply:
Ok, thanks for clarification.